Bot.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[ ]:
+
+
+import telebot
+import ORMForBot as ORM
+from telebot import types
+import TicTacToeForBot as TicTacToe
+bot = telebot.TeleBot('5243687518:AAEYQJGdmCx86OvyIRTzAjDXwN1lM0dgFXU');
```

Comment:
There is a good instrument to work with secret tokens: https://github.com/theskumar/python-dotenv

And a constant string in the middle of the code seems not to be the best solution.